.gitignore

```diff
@@ -16,3 +16,6 @@ munge-csv
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 _output/
+
+.kubeconfig
+.idea
```

Comment:
Just a note, it's likely easier to put stuff that characterizes your personal dev environment into `.git/info/exclude` file